GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit, HostBinding, ViewChild, ElementRef } from '@angular/core';
+import { ChartSettings } from './../dashboard.component';
+
+@Component({
+  selector: 'app-progress-bar',
+  templateUrl: './progress-bar.component.html',
+  styleUrls: ['./progress-bar.component.scss']
+})
+export class ProgressBarComponent implements OnInit {
+  @ViewChild('progressBar') progressBar: ElementRef;
+
+  level = 1;
+  levelString: String;
+  // levelArray = ['Beginner', 'Enthousiasteling', 'Professional', 'Expert', 'Legend'];
+  levelArray = [
+    { 'name': 'Beginner', 'steps': [] },
+    { 'name': 'Enthousiasteling', 'steps': [
+        { 'desc': 'Je Facebook account toe te voegen', 'link': '<>', 'done': true },
+        { 'desc': 'Vandaag iets te posten', 'link': '<>', 'done': false },
+        { 'desc': 'Je Twitter account toe te voegen', 'link': '<>', 'done': false }
+      ] },
+    { 'name': 'Professional', 'steps': [
+      { 'desc': 'Je LinkedIn account toe te voegen', 'link': '<>', 'done': false },
+      { 'desc': 'Deze week twee keer  iets te posten', 'link': '<>', 'done': true },
+      { 'desc': 'Je Hyves account toe te voegen', 'link': '<>', 'done': true }
+    ] },
+    { 'name': 'Expert', 'steps': [] },
+    { 'name': 'Legend', 'steps': [] }
+  ];
+  maxLevels = this.levelArray.length;
+
+  value = 0;
+  v: number;
+  valueString: string;
+  maxValue = 100;
+
```

Comment:
Wederom graag values geven in de constructor, alleen de type boven de constructor definiëren.